src/workers/job-processes.js

```diff
@@ -137,6 +149,7 @@ const processMap = {
 const syncProcessMap = {
   // 'failedMessageSender': failedMessageSender, //see method for danger
   handleIncomingMessages,
+  processSqsMessages,
```

Comment:
I think you want this to be `checkMessageQueue` now

Reply:
oh yea of course